fs/dafsstream/dafsstream.cpp

```diff
@@ -786,8 +808,11 @@ class CDFUPartReader : public CDaFileSrvClientBase, implements IDFUFilePartReade
             endOfStream = true;
             return;
         }
+
+        Owned<IPropertyTree> fetchBatch = getFPosBatch(); // filled if fetching
```

Comment:
Will this overwrite the list of fetch positions returned in the continuation record, if all the fpos haven't been matched?

Reply:
It will add them at the server, but it does highlight a problem.
If reply limit hit, and therefore unconsumed fpos' at server, this will get another 'max' and send.
That means the amount will grow each round.

Need to check amount consumed and send max-consumed.
The cursor being sent back in a fetch has this info, at the moment, the client is agnostic to what the cursor contains, but I'll probably parse it for this info. when fetching.